documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -69,3 +69,9 @@ forklift-controller-788bdb4c69-mw268  2/2    Running  0         2m
 forklift-operator-6bf45b8d8-qps9v     1/1    Running  0         5m
 forklift-ui-7cdf96d8f6-xnw5n          1/1    Running  0         2m
 ----
++
```

Comment:
Wanted to emphasize that there is no relation between possible added provider and possible added mapping.
These are 2 separate things that might have been added before upgrade and if users want to keep on using them,
they'll need to do some steps.

Also why there is a direction to MTV-2.2 doc /OCP-4.9?
This documentation change is for MTV-2.3.

Reply:
Change the text. The OCP version number is controlled by an attribute;  I changed its value.